platform/admin/core/admin-core-configpolicy/src/main/java/org/codice/ddf/admin/config/policy/AdminConfigPolicy.java

```diff
@@ -0,0 +1,170 @@
+
+                if (matchPermissionName.equals(FEATURE_NAME)) {
+                    policyPermissions = featurePolicyPermissions;
+                } else if (matchPermissionName.equals(SERVICE_PID)) {
+                    policyPermissions = servicePolicyPermissions;
+                } else {
+                    //Not a known policy extension key
+                    continue;
+                }
+
+                boolean isPermitted = true;
+
+                //Typically only one feature or service is desired to be permitted at a time but there is support for multiple feature
+                //If there are multiple features in the permission and one is not authorized, the user is not authorized to see any of the features in the group
+                for (String matchPermissionValue : ((KeyValuePermission) permission).getValues()) {
+                    List<KeyValueCollectionPermission> matchOneAttributes = policyPermissions.get(
+                            matchPermissionValue);
+
+                    //Not a known restricted attribute, the feature or service is white listed
+                    if (matchOneAttributes == null) {
+                        continue;
+                    }
+
+                    boolean matchedOneAttribute = false;
+
+                    for (KeyValueCollectionPermission attributePermissions : matchOneAttributes) {
+                        if (subjectAllCollection.implies(attributePermissions)) {
+                            matchedOneAttribute = true;
+                            break;
+                        }
+                    }
+
+                    if (!matchedOneAttribute) {
+                        isPermitted = false;
+                        break;
+                    }
+                }
+
+                if (!isPermitted) {
+                    newMatchCollection.addAll(Collections.singleton(permission));
+                }
+
+            }
+        }
+
+        return newMatchCollection;
+    }
+
+    public void setFeaturePolicies(List<String> featurePolicies) {
+        featurePolicyPermissions.clear();
+        featurePolicyPermissions.putAll(parsePermissions(featurePolicies));
+    }
+
+    public void setServicePolicies(List<String> servicePolicies) {
+        servicePolicyPermissions.clear();
+        servicePolicyPermissions.putAll(parsePermissions(servicePolicies));
+    }
+
+    public Map<String, List<KeyValueCollectionPermission>> parsePermissions(List<String> policies) {
+
+        Map<String, List<KeyValueCollectionPermission>> newPolicyPermissions = new HashMap<>();
+
+        for (String policy : policies) {
+
+            if (StringUtils.isEmpty(policy)) {
+                continue;
+            }
+
+            //Example input: featureName="attributeName=attributeValue","attributeName2=attributeValue2"
+            String policyTrimmed = policy.replaceAll("\\s+", "");
+            String permissionName = policyTrimmed.split("=", 2)[0];
```

Comment:
Instead of calling `split` twice and incurring the cost of the regex matching twice, just call it one time and save the resultant `String[]`. Pull off elements 0 and 1 after.